<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updatemember</class>
 <widget class="QDialog" name="updatemember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Yu Gothic UI&quot;;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lbluMemberName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Yu Gothic UI&quot;;</string>
      </property>
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lbluMemberNumber"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Yu Gothic UI&quot;;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lbluMemberEmail"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnsaveupdatemember">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
